jvm/sargon-android/src/main/java/com/radixdlt/sargon/extensions/PublicKey.kt

```diff
@@ -0,0 +1,36 @@
+package com.radixdlt.sargon.extensions
+
+import com.radixdlt.sargon.BagOfBytes
+import com.radixdlt.sargon.PublicKey
+import com.radixdlt.sargon.ed25519PublicKeyToBytes
+import com.radixdlt.sargon.ed25519PublicKeyToHex
+import com.radixdlt.sargon.newEd25519PublicKeyFromBytes
+import com.radixdlt.sargon.newEd25519PublicKeyFromHex
+import com.radixdlt.sargon.newSecp256k1PublicKeyFromBytes
+import com.radixdlt.sargon.newSecp256k1PublicKeyFromHex
+import com.radixdlt.sargon.secp256k1PublicKeyToBytes
+import com.radixdlt.sargon.secp256k1PublicKeyToHex
+
+fun PublicKey.Ed25519.Companion.init(hex: String): PublicKey.Ed25519 =
```

Comment:
does UniFFI created the nested type `PublicKey.Secp256k1` for you?

Reply:
Yes